public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -554,15 +553,29 @@ export class LokiDatasource
   }
 
   getLogRowContext = (row: LogRowModel, options?: RowContextOptions): Promise<{ data: DataFrame[] }> => {
-    const target = this.prepareLogRowContextQueryTarget(
-      row,
-      (options && options.limit) || 10,
-      (options && options.direction) || 'BACKWARD'
-    );
+    const direction = (options && options.direction) || 'BACKWARD';
+    const limit = (options && options.limit) || 10;
+    const { query, range } = this.prepareLogRowContextQueryTarget(row, limit, direction);
+
+    const sortResults = (result: DataQueryResponse): DataQueryResponse => {
+      return {
+        ...result,
+        data: result.data.map((frame: DataFrame) => {
+          const timestampFieldIndex = frame.fields.findIndex((field) => field.type === FieldType.time);
+          if (timestampFieldIndex === -1) {
+            return frame;
+          }
+
+          return sortDataFrameByTime(frame, 'DESCENDING');
+        }),
+      };
+    };
+
+    // i think this can only be called from explore currently
```

Comment:
```suggestion
    // this can only be called from explore currently
```

Yes, context is available only in Explore.